age/keysource.go

```diff
@@ -96,27 +96,46 @@ func (key *MasterKey) SetEncryptedDataKey(enc []byte) {
 
 // Decrypt decrypts the EncryptedKey field with the age identity and returns the result.
 func (key *MasterKey) Decrypt() ([]byte, error) {
-	ageKeyFilePath, ok := os.LookupEnv("SOPS_AGE_KEY_FILE")
+	var ageKeyReader io.Reader
+	var ageKeyReaderName string
+
+	if ageKeyReader == nil {
+		ageKey, ok := os.LookupEnv("SOPS_AGE_KEY")
+		if ok {
+			ageKeyReader = strings.NewReader(ageKey)
+			ageKeyReaderName = "environment variable"
```

Comment:
I think we should change this to `SOPS_AGE_KEY`, or rather just use the constant used in `os.LookupEnv`

Reply:
@ajvb Good idea 👍 Done